week2/intro_to_property_packages_verbose.ipynb

```diff
@@ -0,0 +1,677 @@
+    "    m1.fs.properties = SeawaterParameterBlock()\n",
+    "    m1.fs.feed = Feed(property_package=m1.fs.properties)\n",
+    "    m1.fs.feed.properties[0].pressure_osm_phase\n",
+    "    m1.fs.feed.properties[0].conc_mass_phase_comp\n",
+    "    m1.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"H2O\"].fix(0.965)\n",
+    "    m1.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"TDS\"].fix(0.035)\n",
+    "    m1.fs.feed.properties[0].temperature.fix(273 + 25)\n",
+    "    m1.fs.feed.properties[0].pressure.fix(101325)\n",
+    "\n",
+    "    print(f\"Degrees of freedom: {degrees_of_freedom(m1)}\\n\")\n",
+    "\n",
+    "    # Initialize and solve for the initial conditions\n",
+    "    m1.fs.feed.initialize()\n",
+    "    results = solver.solve(m1)\n",
+    "    print(f\"Solve termination {results.solver.termination_condition}\")\n",
+    "    # Unfix TDS mass flow\n",
+    "    m1.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"TDS\"].unfix()\n",
+    "\n",
+    "    return m1\n",
+    "\n",
+    "\n",
+    "m1 = build_seawater_prop_model()\n",
+    "\n",
+    "\n",
+    "# =======================================================================================\n",
+    "# Build the model for NaCl property package\n",
+    "def build_NaCl_prop_model():\n",
+    "    \"\"\"\n",
+    "    Create feed model using NaCl property package\n",
+    "    \"\"\"\n",
+    "    m2 = ConcreteModel()\n",
+    "    m2.fs = FlowsheetBlock(dynamic=False)\n",
+    "    m2.fs.properties = NaClParameterBlock()\n",
+    "    m2.fs.feed = Feed(property_package=m2.fs.properties)\n",
+    "    m2.fs.feed.properties[0].pressure_osm_phase\n",
+    "    m2.fs.feed.properties[0].conc_mass_phase_comp\n",
+    "    m2.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"H2O\"].fix(0.965)\n",
+    "    m2.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"NaCl\"].fix(0.035)\n",
+    "    m2.fs.feed.properties[0].temperature.fix(273 + 25)\n",
+    "    m2.fs.feed.properties[0].pressure.fix(101325)\n",
+    "\n",
+    "    print(f\"Degrees of freedom: {degrees_of_freedom(m2)}\\n\")\n",
+    "\n",
+    "    # Initialize and solve for the initial conditions\n",
+    "    m2.fs.feed.initialize()\n",
+    "    results = solver.solve(m2)\n",
+    "    print(f\"Solve termination {results.solver.termination_condition}\")\n",
+    "    # Unfix NaCl mass flow\n",
+    "    m2.fs.feed.properties[0].flow_mass_phase_comp[\"Liq\", \"NaCl\"].unfix()\n",
+    "\n",
+    "    return m2\n",
+    "\n",
+    "\n",
+    "m2 = build_NaCl_prop_model()\n",
+    "\n",
+    "# =======================================================================================\n",
+    "# Define a range of concentrations\n",
+    "low_concs = np.linspace(5, 70, 25)  # g/L\n",
+    "# Create empty lists to store osmotic pressures for each model\n",
+    "p_osm1_low = []\n",
+    "p_osm2_low = []\n",
+    "\n",
+    "\n",
+    "for c in low_concs:\n",
```

Comment:
Consider adding an in-line comment above for-loop saying something like: solves for osmotic pressures across the range of concentrations?

Should also include in skinny version